.travis.yml

```diff
@@ -28,6 +28,7 @@ before_install:
 
 install:
   - make install
+  - conda update -y -c conda-forge geopandas=0.4.1
```

Comment:
this should go in pyam/ci/environment-conda-forge.txt (where you should be able to change the entry to `geopandas<0.5.0`.

Reply:
done